test/api/v01/helpers/request_helper.rb

```diff
@@ -86,6 +86,26 @@ def wait_status_csv(job_id, status, options)
     last_response.body
   end
 
+
+  def wait_solution(job_id, options)
```

Comment:
I've looked at the existing methods, and we could rely on `wait_avancement_match(job_id, "Iteration :", api_key: 'ortools')`